Userland/Libraries/LibWeb/CSS/Properties.json

```diff
@@ -2799,6 +2799,18 @@
       "normal"
     ]
   },
+  "writing-mode": {
+    "animation-type": "none",
+    "inherited": true,
+    "initial": "horizontal-tb",
+    "valid-identifiers": [
+      "horizontal-tb",
+      "vertical-rl",
+      "vertical-lr",
+      "sideways-rl",
+      "sideways-lr"
+    ]
```

Comment:
I'm late, but please use `"valid-types": [ "writing-mode" ]` when you have an enum defined with the same values. You remove some duplication that way.